index.html

```diff
@@ -113,11 +114,28 @@ <h2>About Myself</h2>
         <li class="details"><span class="skills">Skills<img class="arrow" src="./images/arrowright.png" alt=""></span> <img src="./images/SeparatorBottom.png"></li>
      </ul>
     </section>
+    <!-- contact section -->
     <div id="contact">
       <div id="contacttext">
         <h2>Contact me</h2>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. <br>
         I’d love to help with it!</p>
+        <form action="https://formspree.io/f/myyaqdqa" method="post">
+            <ul>
+                <li>
+                    <input type="text" id="name" name="user_name" maxlength="30" placeholder="Enter your name" required>
+                </li>
+                <li>
+                    <input type="email" id="mail" name="user_email" placeholder="Enter your email" required>
+                </li>
+                <li>
+                    <textarea  id="msg" name="user_message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
+                </li>
+                <li class="button">
+                    <button id="butt" type="submit">Get in touch</button>
```

Comment:
- [ ] You have not implemented the styling for pressed & hover state as shown in the button interaction requirements below, you can do this by making use of the css `:active & :hover` pseudo selector. please ensure you implement this following the provided styling on figma for all buttons in your application :+1: :sparkles: 
- [ ] Please ensure you use the appropriate `background color` & `text color` as provided by the figma template design :+1: :sparkles: 
- button interaction
![ca](https://github.com/AdrianScharff/My-portfolio-setup/assets/89055502/7b426732-d88f-43fa-9179-9ca427c74df3)